modules/services/autoraise.nix

```diff
@@ -0,0 +1,160 @@
+              type = nullOr (addCheck float (x: x >= 0 && x <= 1));
+              default = null;
+              description = "A Factor between 0 and 1. Makes the mouse jump vertically to the activated window. By default disabled.";
+            };
+            scale = lib.mkOption {
+              type = nullOr float;
+              default = null;
+              description = "Enlarge the mouse for a short period of time after warping it. The default is 2.0. To disable set it to 1.0.";
+            };
+            altTaskSwitcher = lib.mkOption {
+              type = nullOr bool;
+              default = null;
+              description = "Set to true if you use 3rd party tools to switch between applications (other than standard command-tab).";
+            };
+            ignoreSpaceChanged = lib.mkOption {
+              type = nullOr bool;
+              default = null;
+              description = "Do not immediately raise/focus after a space change. The default is false.";
+            };
+            invertIgnoreApps = lib.mkOption {
+              type = nullOr bool;
+              default = null;
+              description = "Turns the ignoreApps parameter into an includeApps parameter. The default is false.";
+            };
+            ignoreApps = lib.mkOption {
+              type = nullOr (listOf str);
+              default = null;
+              description = "Comma separated list of apps for which you would like to disable focus/raise.";
+            };
+            ignoreTitles = lib.mkOption {
+              type = nullOr (listOf str);
+              default = null;
+              description = "Comma separated list of window titles (a title can be an ICU regular expression) for which you would like to disable focus/raise.";
+            };
+            stayFocusedBundleIds = lib.mkOption {
+              type = nullOr (listOf str);
+              default = null;
+              description = "Comma separated list of app bundle identifiers that shouldn't lose focus even when hovering the mouse over another window.";
+            };
+            disableKey = lib.mkOption {
+              type = nullOr (enum [
+                "control"
+                "option"
+                "disabled"
+              ]);
+              default = null;
+              description = "Set to control, option or disabled. This will temporarily disable AutoRaise while holding the specified key. The default is control.";
+            };
+            mouseDelta = lib.mkOption {
+              type = nullOr float;
+              default = null;
+              description = "Requires the mouse to move a certain distance. 0.0 = most sensitive whereas higher values decrease sensitivity.";
+            };
+            verbose = lib.mkOption {
+              type = nullOr bool;
+              default = null;
+              description = "Set to true to make AutoRaise show a log of events when started in a terminal.";
+            };
```

Comment:
For this option to be useful, the service needs to have `StandardOutPath` set. If you want, you can check whether this option is set and set `StandardOutPath` if so, but it's probably sufficient to just include a note in this description saying that it must be set.

Reply:
In the name of letting the descriptions of these options be identical to upstream, I think you can leave this as is, but should add an option for controlling the log file path like [the ipfs module does](https://github.com/nix-darwin/nix-darwin/blob/master/modules/services/ipfs.nix)